src/main/java/com/redhat/devtools/intellij/knative/kn/KnCli.java

```diff
@@ -83,6 +85,7 @@ private void reportTelemetry() {
         TelemetryMessageBuilder.ActionMessage telemetry = TelemetryService.instance().action(TelemetryService.NAME_PREFIX_MISC + "login");
         try {
             ClusterInfo info = ClusterHelper.getClusterInfo(client);
+            isOpenshift = info.isOpenshift();
```

Comment:
`getClusterInfo` is a costly method that includes possibly several round trips to the cluster. To detect if it's an OpenShift cluster, you should use `client.isAdaptable(OpenShiftClient.class)` instead